packages/compass-e2e-tests/helpers/commands/set-feature.ts

```diff
@@ -17,3 +20,23 @@ export async function setFeature<K extends keyof UserPreferences>(
     value
   );
 }
+
+// When running in Compass web we cannot use the IPC to update the
+// preferences so we use a global function.
+export async function setFeatureCompassWeb<K extends keyof UserPreferences>(
```

Comment:
I think the rule of thumb for e2e to have one method that handles the difference internally instead of having separate methods that you need to explicitly check for when using, this would make these tests more runtime agnostic, allowing us to easier activate them for desktop at the point when we'd be able to run them in the environment